distribution/changelog.txt

```diff
@@ -1,5 +1,6 @@
 0.4.9 (in development)
 ------------------------------------------------------------------------
+- Improved: [#21356] Resize the title bar when moving between displays with different scaling factors on Windows systems.
```

Comment:
The entry looks ok, but there are merge conflicts because your fork is outdated. Please rebase it